pkgs/by-name/za/zarf/package.nix

```
@@ -53,7 +53,7 @@ buildGoModule rec {
   meta = with lib; {
     description = "DevSecOps for Air Gap & Limited-Connection Systems. https://zarf.dev";
     mainProgram = "zarf";
-    homepage = "https://github.com/defenseunicorns/zarf.git";
+    homepage = "https://github.com/zarf-dev/zarf.git";
```

```suggestion
    homepage = "https://zarf.dev";
```
Looks like upstream has a proper homepage.

---

Thank you for the hint. I adapted the attribute accordingly.